Week-1/Homework/2.DanishLetters.js

```diff
@@ -0,0 +1,33 @@
+//find and count danish letters
+
+function danishLetters(string) {
+  //return errrors
+  if (typeof string != "string") {
+    return;
+  }
+
+  string.toLowerCase();
```

Comment:
`string.toLowerCase()` is non-destructive, meaning it doesn't modify the string.

f.x.: const string = 'bLaH'; string.toLowerCase(); console.log(string) --> still prints out `bLaH`

you need to reassign string to `string.toLowerCase()`

Reply:
fixed na mamser